packages/core/useElementHover/directive.ts

```diff
@@ -0,0 +1,15 @@
+import { watch } from 'vue-demi'
+import type { FunctionDirective } from 'vue-demi'
+import { useElementHover } from '.'
+
+type BindingValueFunction = (state: boolean) => void
+
+export const vHover: FunctionDirective<
+HTMLElement,
+BindingValueFunction
+> = (el, binding) => {
+  if (typeof binding.value === 'function') {
+    const isHovered = useElementHover(el)
+    watch(isHovered, v => binding.value(v))
```

Comment:
I think replace `watchEffect` has better performance.
```suggestion
    watchEffect(() => binding.value(unref(isHovered)))
```

Reply:
The watchEffect will triggered once immediately which will give The user unexpected behavior . (triggered `onHover` callback)

Reply:
but it looks like when `isHovered` change then trigger the callback and pass `isHovered`.

Reply:
maybe you want like this?
```typescript
if(isHovered) binding.value(unref(isHovered))
```

Reply:
I think we also need triggered the callback  while the state is `false`